scst-store/multicluster-setup.hbs.md

```diff
@@ -36,7 +36,7 @@ Store in a multicluster setup:
    2. Copy the AMR CloudEvent Handler edit token from the View cluster.
```

Comment:
Not necessarily the right line but just wanted the comment somewhere visible. 

This doc is now all about multi-cluster setup with Scan 2.0, right? 

Is there a place where we tell users how to setup for multi-cluster even though they're (still) using Scan 1.0? Or is this a clever way to get users to change to use Scan 2.0?

Reply:
This is not really specific to 2.0. How you got the impression that it is all about 2.0?

Reply:
I didn't think that Scan 1.0 worked with AMR observer. Perhaps my assumptions was wrong 😅

Reply:
Scan 1.0 still works, to enable Scan 2.0 you have to install `app-scanning-2.0`

Reply:
I had a conversation with Jason yesterday that helped me understand how AMR and AMR observer could co-exist with Scan 1.0. Thanks